wagtail/admin/views/pages/edit.py

```diff
@@ -55,6 +56,45 @@ def add_save_confirmation_message(self):
 
         messages.success(self.request, message)
 
+    def send_commenting_notifications(self):
+        """
+        Sends notifications about any changes to comments to anyone who is subscribed.
+        """
+        # Skip if this page does not have CommentPanel enabled
+        if 'comments' not in self.form.formsets:
+            return
+
+        # Get changes
+        comments_formset = self.form.formsets['comments']
+        new_comments = comments_formset.new_objects
+        deleted_comments = comments_formset.deleted_objects
+
+        # Assume any changed comments that are resolved were only just resolved
+        resolved_comments = []
+        for changed_comment, changed_fields in comments_formset.changed_objects:
+            if changed_comment.resolved_at and 'resolved' in changed_fields:
+                resolved_comments.append(changed_comment)
+
+        # Skip if no changes were made
+        if not new_comments and not deleted_comments and not resolved_comments:
+            return
+
+        # Get subscribers
+        subscribers = PageSubscription.objects.filter(page=self.page, comment_notifications=True).select_related('user')
+        recipient_users = [subscriber.user for subscriber in subscribers if subscriber.user != self.request.user]
```

Comment:
This doesn't account for the userprofile opt out setting

Reply:
This is accounted for in ``send_notification``

Reply:
Oh yep, my mistake, sorry! Was looking for the name explicitly, I see the check now. (Trying to code review in a hurry, oops)